test/hotspot/jtreg/vmTestbase/nsk/jvmti/GetCurrentContendedMonitor/contmon001.java

```diff
@@ -75,13 +83,13 @@ public static int run(String argv[], PrintStream ref) {
             out.println("Check #1 done");
 
         contmon001a thr = new contmon001a();
-        startingBarrier = new Wicket();
-        waitingBarrier = new Wicket();
```

Comment:
If you want to continue to use Wicket, then "all" you have to do is
execute one back-and-forth of the Wicket before executing the
test portion itself. That should get all the Wicket classes loaded
before you really need them. Unless, of course, there's some class
that isn't loaded unless there's an error or something...

However, by using volatile booleans instead, you get out of the
game of trying to figure out how much of the Wicket code needs
to be exercised to get everything you might need loaded.

Reply:
We may load different classes depending on far in method waitFor we have reach before the unlock is called, on both sides.
So unlocked or waitFor may see a new class deeding on if are we added to queue, have we called the parker, etc..
Even if that is easy to figure out, if someone changes the Wicket or e.g. AbstractQueuedSynchronizer, the test may fail again.